### ISSUE TITLE
Wrongly indents comments that follow indented (e.g. conditional) imports

### ISSUE THREAD

**Issue by andersk** [timothycrosley/isort#1088]
isort (commit fbe1ab9290f08521d39a929b516a973b14a53627 from `develop`) adds indentation and a blank line to this comment that it should be leaving alone, since it’s after the `import` statement and outside its block.

```diff
ERROR: /tmp/test.py Imports are incorrectly sorted and/or formatted.
--- /tmp/test.py:before	2020-01-07 16:54:41.150438
+++ /tmp/test.py:after	2020-01-07 16:58:16.519487
@@ -1,5 +1,6 @@
 if True:
     import foo
 
-# comment for bar
+    # comment for bar
+
 bar = "bar"
```

**Comment by timothycrosley** [comment]
Thanks for reporting! I'll make sure to resolve this prior to release

**Comment by timothycrosley** [comment]
This is fixed in develop and will make it's way into 5.0.0 release :)
